com.unity.shadergraph/Editor/Drawing/Inspector/PropertyDrawerUtils.cs

```diff
@@ -12,5 +17,57 @@ public static Label CreateLabel(string text, int indentLevel = 0, FontStyle font
             labelVisualElement.style.unityFontStyleAndWeight = fontStyle;
             return labelVisualElement;
         }
+
+        internal static void AddDefaultNodeProperties(VisualElement parentElement, AbstractMaterialNode node, Action setNodesAsDirtyCallback, Action updateNodeViewsCallback)
```

Comment:
Moved this code here so we could share it between AbstractMaterialNode and CustomFunctionNode.